packages/nextjs/pages/api/stub/globalScore/index.ts

```diff
@@ -4,7 +4,8 @@ import { getGlobalScoreFiltered, getGlobalScoreFilteredDate } from "~~/services/
 import { GlobalScoreDay } from "~~/services/database/schema";
 
 export interface GlobalScoreDTO {
-  [key: string]: string | number;
+  createdAt: string;
```

Comment:
"createdAt" should probably be "date" or something. Unless you want it to actually be the date the record was created.